content/en/apps/examples/covid-education.md

```diff
@@ -0,0 +1,49 @@
+
+* Build on verifiable health safety guidance from the CDC and WHO
+* Can be deployed using the CHT app or SMS workflows 
+* Capture assessment and understanding of CHW knowledge through quizzes, which can be tracked by Supervisors
+
+### Users and Hierarchy Example
+
+| User                             | Location                               | Devices                             | Role                                                                                                                                                                                                                                              |
+| :------------------------------- | :------------------------------------- | :---------------------------------- | :------------------------------------------------------------------------------------------------- |
+| National Officials and County Teams  | Central and district offices           | Desktop, laptop and smartphones     | Monitor central analytics regarding onboarding and training completion. Revisit content where needed.           |
+| Sub-County Teams       | Facilities or local offices            | Tablet or smartphone                | Review CHW progress on aggregate. Support CHW supervisors and monitor location specific analytics.            |
+| CHW Supervisors                  | Community level, based at facilities   | Smartphones and personal phones     | Verify CHW onboarding and training completion. Follow-up with those who have not completed training. Complete their own onboarding and training tasks.                                    |
+| CHWs and Volunteers              | Community level                        | Feature phones and some smartphones | Complete onboarding and training via SMS or App. Provide feedback and ask for support where needed.         |
+
+## Workflow Example
+
+This demo illustrates how a CHT workflow can be easily adapted to integrate a sequence of learning modules through the usage of tasks. These modules are designed to be delivered sequentially to promote engagement and absorption of content. Learning can be tracked across CHWs to inform opportunities to provide additional support. The sequence can also be modified to interact with the CHW through SMS, utilizing 'question and reply' text-based workflows.
+<br />
+
+{{< youtube pFEFIY_SA7M >}}
+
+<br />
+For more detailed information and to download detailed content, please reference the documentation for both the [CHT application](https://docs.google.com/document/d/1Vd6x_WwS-S_sekRQnvW1UAS_rWaTnaawITfeXgX6Sdg) and the [SMS and Messaging workflow](https://docs.google.com/document/d/1fPqZhl5gYoW_UJx6E3EWngWcICOH8mfmslo4r5JqmE0). These showcase the content, key considerations for adapting the content, and the technical implementation. 
```

Comment:
The first link is undershared (can't access) and the second is overshared (anyone with link can edit). Can access be updated accordingly? Also, where appropriate can the content be included here, or link to a published paper?

Reply:
I discussed with Helen and David (original authors / squad leads) and decided to only showcase the SMS document for additional information (access corrected). Also updated the text to better align with the other examples.

Reply:
Ok, I still see both links in the [newest version](https://github.com/medic/cht-docs/blob/1d379cae6d716370fe7e8f0ad6639aaa6c72f972/content/en/apps/examples/covid-education.md#workflow-example), and can't access the [CHT application doc](https://docs.google.com/document/d/1Vd6x_WwS-S_sekRQnvW1UAS_rWaTnaawITfeXgX6Sdg).